grammar.coffee

```diff
@@ -195,6 +205,14 @@ module.exports = grammar
 			choice(@_loop_statement, @if_statement)
 		)
 
+		break_statement: -> prec(PREC.BREAK_STATEMENT, seq('break', optional(@identifier)))
+
+		continue_statement: -> prec(PREC.CONTINUE_STATEMENT, seq('continue', optional(@identifier)))
+
+		fallthrough_statement: -> 'fallthrough'
+
+		return_statement: -> prec(PREC.RETURN_STATEMENT, seq('return', optional(@_expression)))
```

Comment:
I'm sorry; that's a tree-sitter bug. It's interpreting the newline as ubiquitous whitespace instead of as the newline token in your `_statement` rule production. Fixing now...